release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3319,6 +3319,40 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+{product-title} release 4.14.25 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:2789[RHBA-2024:2789] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:2792[RHBA-2024:2792] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.25 --pullspecs
+----
+
+[id="ocp-4-14-25-bug-fixes"]
+==== Bug fixes
+
+* Previously, timeout values larger than what Go programming language could parse were not properly validated. Consequently, timeout values larger than what HAProxy could parse caused issues with HAProxy. With this update, if the timeout specifies a value larger than what can be parsed, it is capped at the maximum that HAProxy can parse. As a result, issues are no longer caused for HAProxy. (link:https://issues.redhat.com/browse/OCPBUGS-30773[*OCPBUGS-30773*])
+
+* Previously, when users imported image stream tags, `ImageContentSourcePolicy` (ICSP) was not allowed to co-exist with `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS). {product-title} ignored any IDMS/ITMS created by the user and favored ICSP. With this release, the image stream tags can co-exist because importing image stream tags now respect IDMS/ITMS when ICSP is also present. (link:https://issues.redhat.com/browse/OCPBUGS-31509[*OCPBUGS-31509*])
+
+* Previously, after you performed an EUS-to-EUS update on your {product-title} cluster that involved pausing and unpausing the machine config pool, two reboot operations occured after the unpause operation. This additional reboot was not expected and was caused by the performance profile controller being reconciled against an older `MachineConfig` that is listed in the `MachineConfigPool`. With this release, the performance profile controller reconciles against the latest `MachineConfig` that is listed in the `MachineConfigPool` so that the extra reboot does not occur. (link:https://issues.redhat.com/browse/OCPBUGS-32980[*OCPBUGS-32980*])
+
+* Previously, a kernel regression that was introduced on {product-title} 4.14.14 caused kernel issues, such as nodes crashing and rebooting, in nodes that mounted to CephFS storage. In this release, the regression issue is fixed so that the kernel regression issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-33251[*OCPBUGS-33251*])
+
+* Previously, `ovs-if-br-ex.nmconnection.\*` files caused the failure of `ovs-configuration.service` and this resulted in nodes getting moved to the `NotReady` state. With this release, `ovs-if-br-ex.nmconnection.*` files are removed from `/etc/NetworkManager/system-connections`, so that this issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-32341[*OCPBUGS-32341*])
```

Comment:
```suggestion
* Previously, `ovs-if-br-ex.nmconnection.\*` files caused the failure of `ovs-configuration.service`, which resulted in nodes being moved to the `NotReady` state. With this release, `ovs-if-br-ex.nmconnection.*` files are removed from `/etc/NetworkManager/system-connections`, so that this issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-32341[*OCPBUGS-32341*])
```